plex.go

```diff
@@ -863,6 +863,35 @@ func (p *Plex) GetLibraries() (LibrarySections, error) {
 	return result, nil
 }
 
+// Retrieve the content inside a library
+func (p *Plex) GetLibraryContent(sectionKey string, filter string) (SearchResults, error) {
+	query := fmt.Sprintf("%s/library/sections/%s/all%s", p.URL, sectionKey, filter)
+
+	resp, err := p.get(query, defaultHeaders())
+
+	if err != nil {
+		return SearchResults{}, err
+	}
+
+	if resp.StatusCode == 401 {
```

Comment:
Not too big of a deal, but we should use the status codes in the http package when doing these checks.